src/pages/resultsView/oncoprint/TracksMenu.tsx

```diff
@@ -60,6 +67,13 @@ export const COUNT_PADDING_WIDTH = 17;
 export default class TracksMenu extends React.Component<IAddTrackProps, {}> {
     @observable tabId: Tab | string = Tab.CLINICAL;
 
+    private genericAssayEntityOptionsReaction: IReactionDisposer;
+
+    private selectedGenericAssayProfileIdByType = observable.map<
```

Comment:
is it possible to have multiple profiles selected?

Reply:
You always have one profile selected per Generic Assay Type: 
![Screenshot from 2023-05-10 12-15-53](https://github.com/cBioPortal/cbioportal-frontend/assets/20278013/164cb92b-aef8-4cab-acca-155cceb6f9c1)

So in this example there are three Generic Assay Types (Generic Assay Patient Test, Treatment Response and Mutational Signature) and both have one profile selected. You cannot have more than one profile selected per Generic Assay Type.